opennms-doc/guide-admin/src/asciidoc/index.adoc

```diff
@@ -133,13 +133,9 @@ include::text/provisioning/single-node.adoc[]
 include::text/provisioning/fine-grained.adoc[]
 include::text/provisioning/api-examples.adoc[]
 
-[ga-business-service-monitoring]
+[[ga-bsm-admin]]
 == Business Service Monitoring
-include::text/bsm/introduction.adoc[]
```

Comment:
introduction.adoc and severity.adoc were removed from the index, but are still present in the source tree